README.md

```diff
@@ -578,6 +579,10 @@ This controls the number of changes processed at once before being sent to the t
 
 This controls the delay in milliseconds between processing each batch of changes.
 
+#### Sql_Trigger_MaxChangesPerWorker
+
+This controls the upper limit on the number of pending changes in the user table that are allowed per application-worker. If the count of changes exceeds this limit, it may result in a scale out. The setting only applies for Azure Function Apps with runtime driven scaling enabled. See the [Scaling](#Scaling) section for more information.
```

Comment:
Change link to lowercase i.e., `#scaling`. If you go to https://github.com/Azure/azure-functions-sql-extension/tree/ameyarele/scalingConfiguration#change-tracking and click on the 'link' icon besides 'Scaling' heading, the URL will have the lowercase fragment name. A different case will work for compatibility reasons, but the convention for link generation is to convert the section heading to lower case and replace the spaces with hyphens.